integration_test/gce/gce_testing.go

```diff
@@ -44,7 +44,8 @@ AGENT_PACKAGES_IN_GCS, for details see README.md.
 
 This library needs the following environment variables to be defined:
 PROJECT: What GCP project to use.
-ZONE: What GCP zone to run in.
+ZONES: What GCP zones to run in, with integer weights attached to each zone,
+in the format: zone1=weight1,zone2=weight2
```

Comment:
Nit: missing trailing '.'.

Reply:
I did this on purpose so that the period wouldn't be misinterpreted as part of the ZONES variable format.